docs/writing-docs/introduction.md

```diff
@@ -4,21 +4,12 @@ title: 'How to document components'
 
 When you write component stories during development, you also create basic documentation to revisit later.
 
-Storybook gives you tools to expand this basic documentation with prose and layout that feature your components and stories prominently. That allows you to create UI library usage guidelines, design system sites, and more.
+Storybook gives you tools to expand this essential documentation with prose and layout that feature your components and stories prominently. That allows you to create UI library usage guidelines, design system sites, and more.
 
-<video autoPlay muted playsInline loop>
-  <source
-    src="addon-docs-optimized.mp4"
-    type="video/mp4"
-  />
-</video>
+![Storybook Docs UI](./storybook-docs-ui-optimized.png)
 
-If you're including Storybook in your project for the [first time](../get-started/install.md), we provide you with [DocsPage](./docs-page.md), a documentation template that lists all the stories for a component and associated metadata. It infers metadata values based on source code, types and JSDoc comments. If you need, you can customize this page to create your own custom template.
+If you're including Storybook in your project for the [first time](../get-started/install.md) or migrating from a previous version, we provide you with a [documentation page](./docs-page.md.md) ("Docs" for short), positioned near your stories. It's a baseline template automatically generated, listing your existing stories and relevant metadata.
```

Comment:
> If you're including Storybook in your project for the [first time](../get-started/install.md) or migrating from a previous version

I don't think this detail is necessary, now that addon-docs has been a default part of SB for so long.